etcd/etcd-env.sh

```diff
@@ -6,11 +6,11 @@ if test x$NAME == x; then
 fi
 export ETCD_NAME=${ETCD_NAME:-$HOSTNAME}
 export ETCD_DATA_DIR=/var/lib/etcd/${NAME}.etcd
```

Comment:
I think the ETCD_DATA_DIR needs to be configurable.

The use case we have is that we want to store the etcd data in an external block device mounted to the host for persistency. Even if we don't want to store etcd's data in an external device we may need to backup etcd.

Thoughts?